sample/AzureAppServicesHostingStartupSample/Startup.cs

```diff
@@ -60,6 +62,13 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
                     await context.Response.WriteAsync(key + ": " + value + Environment.NewLine);
                 }
                 await context.Response.WriteAsync(Environment.NewLine);
+
+                var protectorProvider = context.RequestServices.GetService<IDataProtectionProvider>();
+                var protector = protectorProvider.CreateProtector("Purpose");
+
+                await context.Response.WriteAsync("Protected Query: " + protector.Protect(context.Request.QueryString.Value) + Environment.NewLine);
```

Comment:
This is a pretty strange sample. How does this show that the light-up is working? You can't tell the difference from any other data protection can you?

Reply:
It doesn't show that light up is working, it just forces data protection to run and provision keys which have to get verified manually.

The same situation as with logging- you have to check that log files exist and it's not obvious from page output.